content/en/guides/inference/_index.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: "W&B Inference"
-weight: 8
+weight: 4
```

Comment:
```suggestion
weight: 5
```

Reply:
I'm gonna do 5 here, just because Weave is so key. Models, Weave, Inference seems like the canonical ordering, and is how the cards are ordered on the homepage. At weight `5` it will match that.